seatunnel-core/seatunnel-core-spark/src/main/java/org/apache/seatunnel/SparkStarter.java

```diff
@@ -293,6 +296,24 @@ protected void appendAppJar(List<String> commands) {
         commands.add(Common.appLibDir().resolve("seatunnel-core-spark.jar").toString());
     }
 
+    /**
+     * change file location to get config file on yarn cluster mode
+     */
+    protected void changeFileLocation() {
+        if ( "cluster".equals(this.commandArgs.getDeployMode().getName())) {
```

Comment:
Please use `org.apache.seatunnel.common.config.DeployMode`  replace `cluster` string.

Reply:
Thank you for your suggestion. I have revised and retested the submission

Reply:
Ok, wait workflows run success. @ruanwenjun Hi wenjun, Can you run workflows?